miio/integrations/fan/zhimi/zhimi_miot.py

```diff
@@ -169,7 +169,7 @@ def temperature(self) -> Any:
 
 
 class FanZA5(MiotDevice):
-    mapping = MIOT_MAPPING
+    mapping = MIOT_MAPPING[MODEL_FAN_ZA5]
```

Comment:
```suggestion
    _mappings = MIOT_MAPPING
```
`mapping` is a remnant from the past, and should not be set directly to allow model-based mappings.